components/generic/RightsStatementButton.vue

```diff
@@ -36,6 +36,6 @@
     color: $mediumgrey;
     background: $offwhite;
     border-radius: 0.25rem;
-    padding: 0.625rem;
+    padding: 0.38rem 0.625rem;
```

Comment:
The rights statement button has a slightly larger font-size (16px) than the share and download button (14px). Could you give the rights button the same size?

Reply:
ok